app_grim_reefer/boards/arm/grim_reefer/Kconfig.defconfig

```diff
@@ -0,0 +1,10 @@
+if BOARD_GRIM_REEFER
+
+config BOARD
+	default "grim_reefer"
+
+config SPI_STM32_INTERRUPT
+	default y
+	depends on SPI
+
```

Comment:
Delete

Reply:
fixed in [314d58d](https://github.com/RIT-Launch-Initiative/FSW/pull/52/commits/314d58d64e4811e7a7bd31817aa276b685ae3c5b)